SignalMessaging/environment/OWSSounds.m

```diff
@@ -172,6 +172,8 @@ + (NSString *)displayNameForSound:(OWSSound)sound
             return @"Call Busy";
         case OWSSound_CallFailure:
             return @"Call Failure";
+        case OWSSound_MessageSent:
```

Comment:
Should we increase our LRU sound cache size by 1?

Reply:
Good idea, done.